dgl_sparse/src/matmul.h

```diff
@@ -53,6 +55,75 @@ torch::Tensor SDDMMNoAutoGrad(
     const c10::intrusive_ptr<SparseMatrix>& sparse_mat, torch::Tensor mat1,
     torch::Tensor mat2_tr);
 
+/**
+ * @brief Broadcast the node feature to the edges and then compute
```

Comment:
Let's avoid use the word "node" or "edge" in the context of sparse matrix operators.

Reply:
updated